### ISSUE TITLE
Issue compiling examples/shapes.rs

### ISSUE THREAD

**Issue by whitwhoa** [three-rs/three#220]
Upon attempting to compile the example/shapes.rs file I receive the following message from the compiler:

```
error[E0277]: the trait bound `mint::Quaternion<f32>: std::convert::From<cgmath::Quaternion<f32>>` is not satisfied
  --> src\main.rs:60:18
   |
60 |             mbox.set_orientation(q);
   |                  ^^^^^^^^^^^^^^^ the trait `std::convert::From<cgmath::Quaternion<f32>>` is not implemented for `mint::Quaternion<f32>`
   |
   = help: the following implementations were found:
             <mint::Quaternion<T> as std::convert::From<[T; 4]>>
   = note: required because of the requirements on the impl of `std::convert::Into<mint::Quaternion<f32>>` for `cgmath::Quaternion<f32>`

error[E0277]: the trait bound `mint::Quaternion<f32>: std::convert::From<cgmath::Quaternion<f32>>` is not satisfied
  --> src\main.rs:61:18
   |
61 |             mcyl.set_orientation(q);
   |                  ^^^^^^^^^^^^^^^ the trait `std::convert::From<cgmath::Quaternion<f32>>` is not implemented for `mint::Quaternion<f32>`
   |
   = help: the following implementations were found:
             <mint::Quaternion<T> as std::convert::From<[T; 4]>>
   = note: required because of the requirements on the impl of `std::convert::Into<mint::Quaternion<f32>>` for `cgmath::Quaternion<f32>`

error[E0277]: the trait bound `mint::Quaternion<f32>: std::convert::From<cgmath::Quaternion<f32>>` is not satisfied
  --> src\main.rs:62:21
   |
62 |             msphere.set_orientation(q);
   |                     ^^^^^^^^^^^^^^^ the trait `std::convert::From<cgmath::Quaternion<f32>>` is not implemented for `mint::Quaternion<f32>`
   |
   = help: the following implementations were found:
             <mint::Quaternion<T> as std::convert::From<[T; 4]>>
   = note: required because of the requirements on the impl of `std::convert::Into<mint::Quaternion<f32>>` for `cgmath::Quaternion<f32>`

error[E0277]: the trait bound `mint::Quaternion<f32>: std::convert::From<cgmath::Quaternion<f32>>` is not satisfied
  --> src\main.rs:63:19
   |
63 |             mline.set_orientation(q);
   |                   ^^^^^^^^^^^^^^^ the trait `std::convert::From<cgmath::Quaternion<f32>>` is not implemented for `mint::Quaternion<f32>`
   |
   = help: the following implementations were found:
             <mint::Quaternion<T> as std::convert::From<[T; 4]>>
   = note: required because of the requirements on the impl of `std::convert::Into<mint::Quaternion<f32>>` for `cgmath::Quaternion<f32>`

error: aborting due to 4 previous errors
```

**Comment by kvark** [comment]
You aren't building the `shape.rs` (which is btw tested on CI to build). Your errors come from `src\main.rs`, which I assume you created and copied the content from `shape.rs`. Take a closer look at the dependencies used, version of cgmath in particular.

**Comment by whitwhoa** [comment]
You are absolutely correct! That is what I did, and once I changed the dependency version on cgmath everything functioned as expected. Apologies for opening an issue on this. I had a feeling it was something on my end (as this was the first time I've done anything with rust). Greatly appreciate the assistance, and love the idea of this crate! Would there happen to be another location for asking questions and providing feedback such as a community forum?
